src/models/constants/translation.constants.ts

```diff
@@ -1,5 +1,8 @@
 export const OWNER_TRANSLATION_KEY = 'common.owner';
 export const ADMIN_TRANSLATION_KEY = 'common.admin';
 export const MEMBER_TRANSLATION_KEY = 'common.member';
+export const LEAD_TRANSLATION_KEY = 'common.lead';
+export const HOST_TRANSLATION_KEY = 'common.host';
 
 export type RoleNameKey = typeof OWNER_TRANSLATION_KEY | typeof ADMIN_TRANSLATION_KEY | typeof MEMBER_TRANSLATION_KEY;
+export type AuthorRoleNameKey = typeof LEAD_TRANSLATION_KEY | typeof HOST_TRANSLATION_KEY;
```

Comment:
this does not give enough context to where it's used. `[Comment/Message]Author` gives more context imo.

Reply:
`MessageAuthorRoleNameKey` I like it more too